introduction.md

```diff
@@ -0,0 +1 @@
+Hello!  My name is Chandra Das.  I work at CoinAge DAO.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.